csi/controller.go

```diff
@@ -34,6 +34,7 @@ const (
 	volumeCapabilityMessageNotMultinodeVolume = "Volume is not a multinode volume"
 	volumeCapabilityMessageReadOnlyVolume     = "Volume is read only"
 	volumeCapabilityMessageNotReadOnlyVolume  = "Volume is not read only"
+	defaultCSIVolumeSize                      = uint64(1024 * 1024 * 1024)
```

Comment:
we do have a default volume size in OSD right?
should we just use that constant here instead of declaring in two places?

Reply:
Yeah, absolutely. Do you know where that is?

Reply:
Tried looking for it, couldn't find it so far.
@adityadani would have a better idea

Reply:
As discussed in email, we need a value in OSD. I think this is the right place since only CSI and not OSD supports default size.